.travis.yml

```diff
@@ -1,4 +1,13 @@
+sudo: false
+git:
+  depth: 1
+branches:
+  only:
+    - master
 language: node_js
-
 node_js:
-  - "0.10"
+  - "4.4"
```

Comment:
OK. I feel better with 4.4. Travis will get the latest 4.4.x. I goes with our README.